### ISSUE TITLE
Cannot install module

### ISSUE THREAD

**Issue by rocuonji** [vtt-lair/token-replacer#67]
Trying to install through foundry and get this error "installation failed: k is not defined"
Tried to use the .json link directly and got a similar one : "installation failed: the request to https://github.com/vtt-lair/token-replacer/blob/main/module.json did not provide a valid manifest file."
![image](https://user-images.githubusercontent.com/86148058/122637343-675eec00-d0ee-11eb-942d-fdeb77bc3367.png)

Using foundry version 0.8.7

**Comment by vtt-lair** [comment]
That is peculiar. There was an issue with the manifest file location it seems, which I've updated and the url now works. I'm however still getting the "k is not defined" error when trying to install it via the installation button.

Using the following manifest file works though:
https://github.com/vtt-lair/token-replacer/releases/latest/download/module.json

I was able to install it using that... will try and figure out why the other error is happening.

**Comment by vtt-lair** [comment]
Ok... found the issue. With the previous release I changed the manifest url to go directly to the versioned url instead of the latest url. The problem being that the manifest url in the module.json was still referencing the latest version.

So the url on my release manifest vs what it was in the module.json was different and caused the issue. I've fixed it now and a after refreshing (F5) on Foundry, it will now download using the install button.

Thanks for bringing this to my attention @rocuonji
